kolibri_zim_plugin/assets/src/views/ZimSearchView.vue

```diff
@@ -260,6 +267,7 @@
         "{count, plural, one {{count} result} other {Top {count} results}} for '{query}'",
       noSearchResultsMsg: "No results for '{query}'",
       moreResultsMsg: '{count, plural, one {{count} more result} other {{count} more results}}',
+      redirectedFromMsg: 'Redirected from {{ title }}',
```

Comment:
This is broken, the template system here is with just one curly bracket
```suggestion
      redirectedFromMsg: 'Redirected from { title }',
```

Reply:
Thank you! That's fixed now :)